commit 301d43df3dce21c35818ffdc8cbb9c7b57235984 (refs/changes/63/3963/3)
Author: Mike Zeller <mike.zeller@joyent.com>
Date:   2018-05-16T14:43:51-07:00 (1 year, 5 months ago)
    
    TRITON-390 TRITON-348 breaks setting up fabrics when sdc_underlay_assignment is set to manual

diff --git a/lib/post-setup/fabrics.js b/lib/post-setup/fabrics.js
index 86ffa77..6af80f1 100644
--- a/lib/post-setup/fabrics.js
+++ b/lib/post-setup/fabrics.js
@@ -631,6 +631,7 @@ Fabrics.prototype.checkAssignment = function checkAssignment(cb) {
             return;
         }
         cb(null);
+        return;
     }
 
     self.sdcadm.napi.getNetworkPool(self.data.sdc_underlay_pool,
